src/NearOrg/Notifications/NotificationButton.jsx

```diff
@@ -9,6 +9,7 @@ const handleTurnOn = props?.handleTurnOn;
 const handleOnCancel = props?.handleOnCancel;
 const getNotificationLocalStorage = props?.getNotificationLocalStorage;
 const handleOnCancelBanner = props?.handleOnCancelBanner;
+const mobileView = props?.mobileView;
```

Comment:
We'll need this prop to use in `src/components/navigation/mobile/MobileNavigation.tsx` to prevent hover effect on mobile